cmd/kaf/node.go

```diff
@@ -44,11 +44,11 @@ var nodeLsCommand = &cobra.Command{
 
 		w := tabwriter.NewWriter(outWriter, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
 		if !noHeaderFlag {
-			fmt.Fprintf(w, "ID\tADDRESS\t\n")
+			_, _ = fmt.Fprintf(w, "ID\tADDRESS\tcontroller\t\n")
```

Comment:
`s/controller/CONTROLLER` to be consistent ?

Reply:
agreed!

Reply:
the change has already done
kaf nodes
output as following:

ID     ADDRESS                CONTROLLER   
1001   192.168.210.250:9092   true  

thanks a lot @birdayz @flyingcougar